packages/edit-post/src/components/sidebar/settings-header/index.js

```diff
@@ -11,11 +12,12 @@ import SidebarHeader from '../sidebar-header';
 
 const SettingsHeader = ( { openDocumentSettings, openBlockSettings, sidebarName } ) => {
 	const blockLabel = __( 'Block' );
+	const documentLabel = applyFilters( 'editor.sidebar.settingsHeader.documentLabel', __( 'Document' ) );
 	const [ documentAriaLabel, documentActiveClass ] = sidebarName === 'edit-post/document' ?
 		// translators: ARIA label for the Document sidebar tab, selected.
-		[ __( 'Document (selected)' ), 'is-active' ] :
+		[ `${ documentLabel } ${ __( '(selected)' ) }`, 'is-active' ] :
```

Comment:
Normally we try to avoid concatenation of translatable strings. In another language "(selected)" may need to be before the document label.
What if we use an sprintf?
```
/* translators: 1: document label */
sprintf( __( '%1$s (selected)' ) )
```

Reply:
`sprintf` is definitely preferred over string concatenation.

Reply:
c2d4dc0